### ISSUE TITLE
If colors.tsv defined, anything not in colors.tsv defaulted to gray

### ISSUE THREAD

**Issue by matthewspeir** [maximilianh/cellBrowser#104]
If I don't define a color for something in colors.tsv, why not just default to whatever coloring is used if colors.tsv isn't defined? Basically, I want to define colors for some fields, but not others.

**Comment by maximilianh** [comment]
good point... there are technical reasons for the current behavior but
technical reasons are rarely good reasons...

**Comment by matthewspeir** [comment]
Haha, makes sense. 

My use case for this is the Mouse Gastrulation dataset: https://hgwdev.gi.ucsc.edu/~mspeir/cb_demo/?ds=mouse_gastrulation. I was able to extract celltype coloring from the metadata and I was able to use the OSX Color Meter tool to find the colors of metadata fields with few (<10) values (Stage, Theiler) on the website (https://marionilab.cruk.cam.ac.uk/MouseGastrulation2018/), but wasn't able to do anything for fields that didn't have colors in the metadata and didn't want to spend the time using the Color Meter for those fields with lots of values (cluster, cluster.theiler, etc). Because I couldn't get colors for these other fields, they now just default to gray.
